etc/kayobe/kolla.yml

```diff
@@ -116,10 +116,19 @@ kolla_build_blocks:
   sahara_engine_footer: |
     # Bring in hack to the vanilla plugin
     RUN pip install -U --no-deps git+https://github.com/stackhpc/sahara.git@stackhpc/7.0.2.1
+  monasca_api_footer: |
+    # NOTE(brtknr): Install upstream patch currently under review to
+    # support time range on the server side
+    RUN pip install -U --no-deps git+https://github.com/stackhpc/monasca-api.git@stackhpc/7.0.2.3
```

Comment:
Please see Mark's project for info on naming the tags: https://github.com/stackhpc/naming-things-is-hard

We should remove this tag and retag. 

You can get the rocky tag from here: https://github.com/openstack/releases/blob/master/deliverables/rocky/monasca-api.yaml

So in this case the tag is: `stackhpc/2.7.0.2` because `stackhpc/2.7.0.2` already exists.

If the project doesn't have an openstack release, then you can just bump the existing version. For example, the datasource: 1.3.0